src/namings.rs

```diff
@@ -80,7 +80,7 @@ pub fn number_to_greek(n: u8) -> String {
                 + "conta"
         }
     } else {
-        panic!(n.to_string().to_owned() + " uncalculatable");
+        panic!("{} uncalculatable", n.to_string().to_owned());
```

Comment:
On second thought this doesn't need `.to_owned()` anymore. Maybe it's even nicer to `impl std::fmt::Display` for whatever type `n` is here and skip the `.to_string()` too

Reply:
i think you're right, ill go ahead and add that to the todo list now